lib/services/morea_firestore.dart

```diff
@@ -8,103 +9,147 @@ import 'package:random_string/random_string.dart' as random;
 
 abstract class BaseMoreaFirebase {
   String get getDisplayName;
+
   String get getPfandiName;
+
   String get getGroupID;
+
   String get getVorName;
+
   String get getNachName;
+
   String get getPos;
-  String get getEventID;
+
+  String get getHomeFeedMainEventID;
+
   String get getEmail;
+
   List<String> get getSubscribedGroups;
-  Map<String,dynamic> get getGroupMap;
-  Map<String,dynamic> get getUserMap;
-  Map<String,Map<String,String>> get getChildMap;
+
+  Map<String, dynamic> get getGroupMap;
+
+  Map<String, dynamic> get getUserMap;
+
+  Map<String, Map<String, String>> get getChildMap;
 
   Future<void> createUserInformation(Map userInfo);
+
   Future<void> updateUserInformation(String userUID, Map userInfo);
+
   Future<DocumentSnapshot> getUserInformation(String userUID);
+
   Stream<QuerySnapshot> getChildren();
+
   Future<void> uploaddevtocken(
-  var messagingGroups, String token, String userUID);
+      var messagingGroups, String token, String userUID);
+
   Stream<QuerySnapshot> getMessages(String groupnr);
+
   Future<void> setMessageRead(String userUID, String messageID, String groupnr);
+
+  Stream<QuerySnapshot> streamCollectionWerChunnt(String eventID);
 }
 
 class MoreaFirebase extends BaseMoreaFirebase {
   CrudMedthods crud0;
   Auth auth0 = new Auth();
   DWIFormat dwiformat = new DWIFormat();
   TeleblizFirestore tbz;
-  Map<String,dynamic> _userMap, _groupMap;
-  Map<String, Map<String,String>> _subscribedGroupsMap, _childMap;
-  String _displayName, _pfadiName, _groupID, _vorName, _nachName, _pos, _eventID, _email;
-  Map<String,dynamic> _messagingGroups;
+  Map<String, dynamic> _userMap, _groupMap;
+  Map<String, Map<String, String>> _subscribedGroupsMap, _childMap;
+  String _displayName,
+      _pfadiName,
+      _groupID,
+      _vorName,
+      _nachName,
+      _pos,
+      _homeFeedMainEventID,
```

Comment:
Das sollte eine Liste sein (List<String>)

Reply:
Was sollte eine Liste sein?

Reply:
_homeFeedMainEventID oder kannst du mir erklähren für was das gebraucht wird?

Reply:
achso...
`_homeFeedMainEventID `war vorher `_eventID`, was auch ein String war keine Liste. Dieser String `_eventID` hatte keine Funktion und ihm wurde kein Wert übergeben. Ich habe diesen String dann umbenannt in `_homeFeedMainEventID`, dem der ID des ersten Events im homeFeed übergeben wird. Dadurch wird dann bei diesem Event eingelesen wer kommt und wer nicht.

Reply:
wenn du dies als Liste haben willst müssten wir einiges ändern gehen und ich denke dass sollte dann als neues Issue aufgemacht werden.

Reply:
oki